ruby/vampires.rb

```diff
@@ -0,0 +1,61 @@
+puts "How many applicants will be processed?"
+	applicants=gets.chomp.to_i
+until applicants==0
+#Survey for incoming applicants
+puts "What is your name?"
+	name=gets.chomp
+if name=="Drake Cula" || "Tu Fang"
+	name=false
+else name=true
+end 
+puts "How old are you?"
+	stated_age=gets.chomp.to_i
+
+puts "What year were you born?"
+	birthday=gets.chomp.to_i
+
+current_year= 2017
```

Comment:
Check out the Ruby Time class and see what you can do with `Time.now.year`.  This will help the code run a but more dynamically.  Otherwise you will have to keep changing this file every new year to keep the code running properly.